src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +61,28 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// function to generate application details content
+const dynamicRowGeneration = (rowElementMaxCount, appDetailLabels, loading) => {
+  const labelArray = appDetailLabels.map(label => {
+    // only generate elements that are populated
+    if (!isEmpty(label[0])) {
```

Comment:
Completely forgot to explain this aspect of booleans, so sorry about this. Lodash's `isEmpty` works by checking to see if if there are enumerable keys (Object) or if there is a length attribute associated with the item (arrays, strings). Booleans have neither, so isEmpty will return false for any boolean. You'll have check on booleans separately from the rest.